src/meson.build

```diff
@@ -228,6 +228,14 @@ mod_rust.test(
         suite: 'unit',
 )
 
+if meson.version().version_compare('>=1.8')
+        mod_rust.doctest(
+                'rbus-doctests',
+                rust_bus,
+                suite: 'unit',
```

Comment:
I suggest using a separate suite here. I'm not sure how your CI works, but some projects do not copy over the individual .o files from the `build` stage to the `test` stage. That may prevent you from running the doctests in the `test` stage, and having a separate suite helps moving them to the `build` stage.

Reply:
Hmmm, `rustdoc` does not allow to split the build stage from the test stage, does it? So you have to call `rustdoc --test [...]` at the time of running the tests, which then needs access to the crate it documents, as this is usually linked into the doc-tests. I see.

I am kinda leaning towards just making doctests build-time optional, then it is easy to disable when anyone runs into such problems.

But putting it into a separate suite is definitely nice, so it can be easily deselected, given its runtime requirements. Thanks for the hint!

Reply:
> Hmmm, rustdoc does not allow to split the build stage from the test stage, does it? 

Yes that's the issue. I meant build and test stage as CI concepts (I'm more familiar with Gitlab but GitHub should have something similar).

> I am kinda leaning towards just making doctests build-time optional, then it is easy to disable when anyone runs into such problems.

Yes that also makes sense and would also be a workaround for any issues with cross toolchain files.

Reply:
Sounds good, thanks a lot!